cmd/crc/cmd/status.go

```diff
@@ -195,11 +195,15 @@ func (s *status) prettyPrintTo(writer io.Writer) error {
 		{"CRC VM", s.CrcStatus},
 	}
 
-	if s.Preset == preset.OpenShift {
+	switch s.Preset {
+	case preset.OpenShift:
 		lines = append(lines, line{"OpenShift", openshiftStatus(s)})
-	}
-	if s.Preset == preset.Microshift {
+	case preset.Microshift:
 		lines = append(lines, line{"MicroShift", openshiftStatus(s)})
+	case preset.OKD:
+		lines = append(lines, line{"OpenShift/OKD", openshiftStatus(s)})
+	default:
+		panic(fmt.Sprintf("unknown preset provided : %s (expecting OpenShift, MicroShift, or OKD) ", s.Preset))
```

Comment:
Also this change is only on cmd side, did you check this looks similar when get this info using `/status` api call using daemon?

Reply:
On crc daemon side, I see that we default to OpenShift preset :

https://github.com/crc-org/crc/blob/1af16203e10253dcb829a9d62fa45bc03f7f4c02/pkg/crc/machine/status.go#L59-L66

Shall I update cmd side logic to make it consistent with this?

Reply:
I've updated it to align with `/status` api call logic.